front-end/src/components/Admin/index.js

```diff
@@ -9,7 +9,7 @@ let index = 1;
 class Admin extends Component {
   state = {
     tabs: [{
-      title: 'Title',
+      title: `${localStorage.lang === 'en' ? 'New tab' : 'Новая вкладка'}`,
```

Comment:
This doesn't look right. Strings in our UI have to be in one language (English is recommended). react-intl have to detect the current language and provide a translation from the language map. The idea of having i18n is that we have all translations in one place.